apps/nextcloud/templates/services/nginx/route.yml.j2

```diff
@@ -0,0 +1,27 @@
+apiVersion: v1
+kind: Route
+metadata:
+  namespace: "{{ project_name }}"
+  name: "nextcloud-nginx-{{ prefix }}"
+  labels:
+    env_type: "{{ env_type }}"
+    customer: "{{ customer }}"
+    app: "nextcloud"
+    service: "nginx"
+    version: "{{ nextcloud_nginx_image_tag }}"
+    route_prefix: "{{ prefix }}"
+    route_target_service: "app"
+  annotations:
+    kubernetes.io/tls-acme: "true"
+    haproxy.router.openshift.io/disable_cookies: "true"
+    haproxy.router.openshift.io/balance: "roundrobin"
```

Comment:
This isn't the default?

Reply:
`¯\_(ツ)_/¯`

Reply:
`ROUTER_TCP_BALANCE_SCHEME for passthrough routes. Otherwise, use ROUTER_LOAD_BALANCE_ALGORITHM.`
from https://docs.openshift.com/container-platform/3.11/architecture/networking/routes.html#route-specific-annotations
Deal with it